analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.CSharp6.cs

```diff
@@ -119,7 +119,8 @@ void NonCompliant2()
       switch (o?.GetHashCode())
       {
         case null:
-          o.ToString(); // FIXME Non-compliant
+          o.ToString(); // Noncompliant
+                        // Noncompliant@-1 FIXME: double reporting
```

Comment:
Seems to be related to switch branching. Using `PostProcessSimple` does not change behavior here.